### PR TITLE
SI methods to `ci()` and `describe_posterior()`

### PR DESCRIPTION
**Pull request by mattansb**
@DominiqueMakowski @strengejacke please take a (quick) look at the implementation (:

### CODE REVIEW

File: R/si.R [hunk 1]
```diff
@@ -132,14 +132,16 @@ si.stanreg <- function(posterior, prior = NULL,
                        BF = 1, verbose = TRUE,
```
Comment (strengejacke) [inline R/si.R]: Looks all good to me! Was just wondering if we should name the argument `BF`, or `bayesfactor`, or both (alias)?

Comment (mattansb) [reply R/si.R]: Having `bayesfactor` might be more confusing than it already is, as this "BF" is not explicitly ***the*** Bayes factor, but a Bayesian (updating) factor. So I think keeping it as is (`BF`) is best, which also matches the terminology in the SI paper (where the support level is simply rendered as *BF*).

Comment (strengejacke) [reply R/si.R]: ok, sound feasible!

Comment (mattansb) [reply R/si.R]: ~So what you're saying is that `F_to_eta2` should be `F_to_eta2_partial`?~
👍

**Pull request merged**